### DISCUSSION TITLE
[자료구조 5일차] 해시

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#34] [오늘의 주제]
해시 관련 질문 2인조 1개, 3인조 2개씩 총 5개

**Comment by sikutisa** [comment]
해시 충돌과 그 해결법에 대해 설명해주세요.

**Comment by sikutisa** [comment]
Direct Addressing Table과 Hash Table에 대해 설명해주세요.

**Comment by watty-exclamationMark** [comment]
자바에서 Hash를 사용하는 자료구조중 HashTable과 HashMap이 있습니다.
둘의 차이점은 무엇일까요?

**Comment by zini9188** [comment]
단방향 해쉬 함수의 문제점을 설명하고 이를 해결하기 위한 방법을 알려주세요

**Comment by watty-exclamationMark** [comment]
해시테이블을 사용하다 보면, 데이터가 고루 분포되지 않고 한 구역에 집중되는 클러스터 현상이 발생할 수 있습니다.
이를 완화하기 위한 방법은 어떤 것이 있을까요?

**Comment by kjc6735** [comment]
해시 테이블과 시간 복잡도에 대해 설명해주세요
